### ISSUE TITLE
Registration fails at creating personal Team with PostgreSQL

### ISSUE THREAD

**Issue by thekiharani** [laravel/jetstream#778]
- Jetstream Version: 2.2.4
- Jetstream Stack: Livewire
- Uses Teams: yes
- Laravel Version: 8.40.0
- PHP Version: 8.0.5
- Database Driver & Version: PostgreSQL 12.6

### Description:
An attempt to register using the default setup results in the following error:

```
SQLSTATE[42804]: Datatype mismatch: 7 ERROR: column "personal_team" is of type boolean but expression is of type integer LINE 1: ...eam", "updated_at", "created_at") values ($1, $2, $3, $4, $5... ^ HINT: You will need to rewrite or cast the expression. (SQL: insert into "teams" ("user_id", "name", "personal_team", "updated_at", "created_at") values (1, Kimberly's Team, 1, 2021-05-02 19:21:22, 2021-05-02 19:21:22) returning "id")
``` 

Using MySQL seems to work so I suspect this is an issue with PostgreSQL or something

### Steps To Reproduce:
- Set PostgreSQL as database driver
- Run migrations
- Register a new user account

**Comment by driesvints** [comment]
This is a bug in PHP. Please see https://github.com/php/php-src/pull/6801

**Comment by njoguamos** [comment]
I was able to fix this by downgrading PHP 8.0.5 to PHP 8.0.3_1 via brew.

Be aware that this will expose your app to the security fix on PHP 8.0.5.

For those interested, [follow this instruction](https://gist.github.com/sawant/6ea10e45de7d09e966a1dc6afea3bffb)

I will upgrade to PHP 8.0.6 once it is released.
